### ISSUE TITLE
Compile tensorflow on OSX with SSE/AVX/FMA ?

### ISSUE THREAD

**Issue by daschl** [tensorflow/rust#58]
Hi,

since the 1.0 upgrade `cargo test` now spits out the following on my MBP:

```
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.1 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX2 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use FMA instructions, but these are available on your machine and could speed up CPU computations.
```

I guess we could be more intelligent when compiling tensorflow to try and take advantage of it automatically?

**Comment by daschl** [comment]
PR #59 should handle this.

**Comment by adamcrume** [comment]
Merged #59.
